jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -1184,6 +1197,15 @@ def _load_regions(self, regions, edit_subset=None, combination_mode=None, max_nu
 
         if not isinstance(regions, (list, tuple, Regions, SpectralRegion)):
             regions = [regions]
+        if isinstance(subset_label, str):
+            subset_label = [subset_label]
+
+        bad_labels = []
+        for label in subset_label:
+            if not self.app._check_valid_subset_label(label, raise_if_invalid=False):
```

Comment:
`app._check_valid_subset_label` seems to raise errors if there is no space due to `if split_label[0] == "Subset" and split_label[1].isdigit():`, can we make that more robust here?

Reply:
and it also might be nice if "Subset N" is allowed at least if it would be the automatic label anyways